python-cppcolormap/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-cppcolormap" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "1b6e0213454e6a7432281224841466363c330d0c343ac1cbea4640c1be77fa27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/cppcolormap/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py build
+    - python setup.py install
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
```

Comment:
You don't use `pip` so it can be removed here?

Reply:
You're right, thanks!